src/neovim/neovim.ts

```diff
@@ -14,8 +15,9 @@ export class Neovim implements vscode.Disposable {
   private nvim: Nvim;
 
   async initialize() {
+    const dir = dirname(vscode.window.activeTextEditor!.document.fileName);
```

Comment:
I think what you want is this `vscode.window.activeTextEditor!.document.uri.fsPath`

Reply:
In the documentation, it said that this is 
```
		/**
		 * The file system path of the associated resource. Shorthand
		 * notation for [TextDocument.uri.fsPath](#TextDocument.uri). Independent of the uri scheme.
		 */
```
so I didn't think there was a difference.

Reply:
Hm, that's confusing. Generally, I would understand things named as `fileName` to be just the file name (`source.ts`), whereas `filePath` would be a file path (`/my/dir/source.ts`)